tests/travis.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
Hmm, in researching what you said I came across https://docs.travis-ci.com/user/customizing-the-build#Implementing-Complex-Build-Steps

I think what I would like to do is `git mv tests/travis.sh tests/run-tests.sh`. I don't see the point of having a `.travis` dir just for that file, and more importantly, that file is not travis specific. Could be used by any of our CICD systems.